presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -288,16 +287,16 @@ public static ColumnMapping timeColumnMappingUsingSqlTime(ConnectorSession sessi
                 TIME,
                 (resultSet, columnIndex) -> {
                     Time time = resultSet.getTime(columnIndex);
-                    return NANOSECONDS.toMillis(toLocalTime(time).toNanoOfDay());
+                    return NANOSECONDS.toMillis(fromSqlTime(time).toNanoOfDay());
                 },
                 timeWriteFunctionUsingSqlTime(session));
     }
 
-    private static LocalTime toLocalTime(Time sqlTime)
+    private static LocalTime fromSqlTime(Time sqlTime)
```

Comment:
This name is closer to the related `toSqlTime`, and in line with the other translation utilities, which all leave the type from `java.time` out of the name.

Reply:
the name was chosen deliberately, not incidentally.
The reason name `toSqlTime(LocalTime)` was picked is that the signature allows to avoid any future conflicts, if we want to have other time representation.
This doesn't apply to `fromSqlTime(Time)`.